Application/Source/Components/TripTable/TripTableRootView.swift

```diff
@@ -10,21 +10,18 @@
 import Reactant
 import RxSwift
 
-final class TripTableRootView: PlainTableView<TripCell> {
-    
-    override var edgesForExtendedLayout: UIRectEdge {
-        return []
-    }
+final class TripTableRootView: ViewBase<[Trip], PlainTableViewAction<TripCell>> {
+
+    let tableView: PlainTableView<TripCell> = PlainTableView(cellFactory: TripCell.init, reloadable: false)
 
-    init() {
-        super.init(
-            cellFactory: TripCell.init,
-            reloadable: false
-        )
+    override var actions: [Observable<PlainTableViewAction<TripCell>>] {
+        return tableView.actions 
+    }
 
+    override func afterInit() {
         tableView.rowHeight = TripCell.height
         tableView.separatorStyle = .none
-        tableView.tableFooterView = UIView() //this line makes searchbarwrapper disappear
-        tableView.contentInset.bottom = 8
+        tableView.tableView.tableFooterView = UIView() //this line makes searchbarwrapper disappear
```

Comment:
You can delete the comment, I put it there back then when there was a bug in Reactant which caused the header to disappear upon setting the `tableFooterView`.